### PR TITLE
WIP: Add standard decorators docs

### PR DESCRIPTION
**Pull request by justinfagnani**
Rendered: https://pr1176-bd411fa---lit-dev-5ftespv5na-uc.a.run.app/docs/v3/components/decorators/

### CODE REVIEW

File: packages/lit-dev-content/site/docs/v3/components/decorators.md [hunk 1]
```diff
@@ -11,9 +11,16 @@ versionLinks:
...
 
-See the [Enabling decorators](#enabling-decorators) section for more information.
+Lit provides two implementations of decorators:
+
+* **Standard Decorators**: Standard decorators are the Stage 3 specification that has been agreed up by the JavaScript standards body (TC39), browsers, and JavaScript compilers. This is the version that browsers will implement, and the ecosystem will migrate to over time. TypeScript and Babel have new implementations of this specification.
+* **Legacy Decorators**: Legacy decorators implement a previous version of the specification, but are currently the most popular and best supported version of the specification.
+
+Because the implementations of "standard decorators" are new, this documented focuses on what we're calling "legacy" decorators, which we will refer to here as just "decorators". Future versions of Lit will migrate to standard decorators as the default.
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/components/decorators.md:21]: "this documented" -> "this document"

File: packages/lit-dev-content/site/docs/v3/components/decorators.md [hunk 2]
```diff
@@ -168,3 +175,26 @@ constructor() {
...
+
+* The emit is large
+* Requires TypeScript 5.2 or Babel (which plugin)
+* Can't mix legacy and standard decorators
+
+### Importing
+
+### Difference from legacy decorators
+
+The standard decorators are a breaking change from the legacy decorators. Partially because of changes required by the new spec, like `accessor` and partially because the new spec allows us to implement behavior that wasn't possible before.
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/components/decorators.md:193]: This paragraph deviates from the H3, please reintroduce the list. e.g. `The differences are:`

File: packages/lit-dev-content/site/docs/v3/components/decorators.md [hunk 3]
```diff
@@ -168,3 +175,26 @@ constructor() {
...
+
+* Import from `lit/std-decorators.js`, etc
+* Reactive properties require the `accessor` keyword
+* Setters are wrapped with a setter than calls `requestUpdate()` for you? (todo)
+* You can decorate private fields
+* queryAsync is removed
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/components/decorators.md:199]: queryAsync -> `queryAsync`

File: packages/lit-dev-content/site/docs/v3/components/decorators.md [hunk 4]
```diff
@@ -168,3 +175,26 @@ constructor() {
...
+* Requires TypeScript 5.2 or Babel (which plugin)
+* Can't mix legacy and standard decorators
+
+### Importing
+
+### Difference from legacy decorators
+
+The standard decorators are a breaking change from the legacy decorators. Partially because of changes required by the new spec, like `accessor` and partially because the new spec allows us to implement behavior that wasn't possible before.
+
+* Import from `lit/std-decorators.js`, etc
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/components/decorators.md:195]: I think people care more about "what does this mean for me?" Here can we emphasize the migration for users? idk if that should be a before / after table or just sub bullets here or maybe a different structure altogether

File: packages/lit-dev-content/site/docs/v3/components/decorators.md [hunk 5]
```diff
@@ -11,9 +11,16 @@ versionLinks:
...
 Decorators are special functions that can modify the behavior of classes, class methods, and class fields. Lit uses decorators to provide declarative APIs for things like registering elements, reactive properties, and queries.
 
-Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Currently no browsers implement decorators, but compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for an earlier version of the decorators proposal. Lit decorators work with Babel and TypeScript, and will be updated to work with the final specification when it's implemented in browsers.
+Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Currently no browsers implement decorators, but compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for standard decorators, as well as an earlier version of the decorators proposal.
 
-See the [Enabling decorators](#enabling-decorators) section for more information.
+Lit provides two implementations of decorators:
```
Comment (e111077) [inline packages/lit-dev-content/site/docs/v3/components/decorators.md:16]: I think users care about "what does this mean for me". This is excellent information and it should stay, but we should maybe add a section of "which decorators should I use for my project? Should I be migrating?"

**Review by e111077 — CHANGES_REQUESTED** [review]

**Comment by AndrewJakubowicz** [comment]
Noting down that this document will have to be updated to the hybrid decorators implementation: Where our decorators work in all environments.

**Comment by justinfagnani** [comment]
Superseded by https://github.com/lit/lit.dev/pull/1218

**Pull request closed without merging**